app/models/record.rb

```diff
@@ -1,5 +1,70 @@
+# frozen_string_literal: true
+
 class Record < ApplicationRecord
   audited only: :updated_at
+  before_save :convert_maz_json_to_sva_json
+  after_save :send_sva_json_to_server
+
+  private
+
+  def convert_maz_json_to_sva_json
+    news_data = parse_single_news_from_json(self.json_data)
+    self.sva_json_data = { news_item: news_data }
```

Comment:
bei den STI-.Modewls hatte ich Schwierigkeiten mit dem setzen der lokalen Attributen und da als workaround `self.` verwendet. Hier könnte das auch ohne `self.` funktionieren. Bitte einmal testen.

Reply:
was bedeutet eigentlich `sva? in `self.sva_json_data`?

Reply:
sva = smart village app